alchemist-full/build.gradle.kts

```diff
@@ -145,6 +146,42 @@ tasks.register<Exec>("testJpackageOutput") {
     finalizedBy(deleteJpackageOutput)
 }
 
+val generatePKGBUILD by tasks.registering(Exec::class) {
+    group = "Publishing"
+    description = "Generates a valid PKGBUILD by replacing values in the template file"
+    val inputFile = file("${project.projectDir}/PKGBUILD.template")
+    val outputDir = file(rootProject.layout.buildDirectory.map { it.dir("package") }.get().asFile.path)
+    val tokenToReplace = "{%}"
+    var replacementValues: List<String>
+    val testSourceParam = System.getProperty("generatePKGBUILD.testSource", "false")
+    val interceptOutput = ByteArrayOutputStream()
+    standardOutput = interceptOutput
+    doFirst {
+        val version = rootProject.version.toString().substringBefore('-')
+        commandLine("md5sum", "-b", "${rootProject.layout.buildDirectory.get().asFile.resolve("package").path}/${rootProject.name}-$version-1.x86_64.rpm")
```

Comment:
we can compute the md5hash in kotlin directly, without going through the command line